Feature/adolescents update v3

Daniel-Gomez-jpg opened the pull request:
adolescents and adults pages updated to v3

Daniel-Gomez-jdk committed:
adolecent page update

Daniel-Gomez-jdk committed:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/adolescents-update-v3

HackMort committed:
Update Page Content & Fix issue on desktop view.

Daniel-Gomez-jdk committed:
adolescent and adult pages updated to v3

Daniel-Gomez-jdk committed:
reference component added to adolescents page

Daniel-Gomez-jdk committed:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/adolescents-update-v3

Daniel-Gomez-jdk committed:
hypo page content updated

Daniel-Gomez-jdk committed:
hero colors and heading colors updated on adults page

Daniel-Gomez-jdk committed:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/adolescents-update-v3

Daniel-Gomez-jdk committed:
adults graphics updated

Commit:
style: :lipstick: Changes for Skeleton Figma v3

Change of content, positions of points, direction of lines before/after click, position of footnote and position of symptoms list

Commit:
feat: :construction: Update of content to Figma v3 Hypo page

Update of super indexes, list item in neonates tabs, texts in highlights on children tab and add class for CTA

Commit:
style: :lipstick: Fix styles for Hypo Page

Add CTA adjustments, margins on first section, padding right/left of tabs in first section, color of footnote on highlight information components for hypo page, and testimonial shape adjustmen on bigger screens'

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/adolescents-update-v3

Daniel-Gomez-jdk committed:
Merge branch 'feature/adolescents-update-v3' of https://github.com/HackMort/strensiq-hcp into feature/adolescents-update-v3

Daniel-Gomez-jdk committed:
new graphics added to adolescents page

HackMort reviewed (changes requested):
Adolescents:

Hero Mobile Shape it does not look the same as the Figma:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/7f28daa8-2ca9-44da-979f-b94ead7ef766)
Checkout spacing of the chart image, position of the pinch to zoom icon, remove the text that displays when you are in the zoom overlay:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/66e05ed7-9ce9-484c-a4a6-018e2d44647f)
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/6a9efb71-c237-4a0e-b34b-aab17ce2a530)
Looks like we have a break line in this section:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/6047a63d-fa9a-4935-bf4a-9ca2a7c6a658)
In this section, the text inside the parenthesis it should be a gray color, not green:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/50d700e3-9637-4998-8e57-03800534ad68)
Match spacing between the chart and the CTA like in the Figma:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/eb846adb-9ee4-48ba-ae5b-50525220ff48)

================================
Adults:
Hero shape is not looking like in the Figma:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/53bb523c-8aa1-4d65-9a98-d78b599ed085)
Bullets should be aligned with the rest of the content:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/4f1bf7c7-7d79-455f-881a-58b42f2fdb85)
Tabs & Heading needs to match the Figma, also the chart seems too small compared to Figma
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/1d340fae-ceb2-4613-85df-96950ee9484a)
Revisit styles:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/0b20abae-0d08-4fb3-8a78-8bfbe625700a)
Add more space below the video placeholder image:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/0e3b19ac-c283-42dc-81ed-072ad57d0ca7)
Adjust graphic, maybe we need to re-export it, and also fix the bullets, they should be aligned with the rest of the content:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/782e198e-365b-45ed-826b-4ae789f38cec)
Please double check this section, spacing, font size, colors, etc:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/16e82786-3f0b-4630-9646-ee0dfa339f8a)
Double check styles:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/75f79f2c-66f4-45f6-9e5d-30d479903352)
The green line below the table header does not seems to be the same height:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/dcd7d4a9-475c-4dac-903c-364b50154c35)
Tab should go all the way to the edge of the browser. Also, please make sure we hide the scrollbars in tabs:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/1a88f937-5d2e-4283-9439-762a140b6edf)
Update styles and graphic:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/ea81e12e-f512-464e-afe3-7fc0cab66467)
Double check graphic and cta styles:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/9b3f76fb-0839-40e7-802f-d2f2ac2a7f02)

Also, don't forget to update the table in the modal if necessary.
================================
Hypophosphatasia:

Please update the heading to not have the superscripts alone in the same line, break it in 2-3 lines like in the FIgma if possible.
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/39d433c8-6519-4719-af15-2bdb5842fa33)
Please double check the spacing in these sections:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/03f2e8a5-92d8-4dbe-9589-3a1ceedb975b)
Checkout tabs style:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/1c689211-74ec-4a27-9324-044caecb7831)
I see a couple of things that look different from the Figma, spacing, alignment and the pink shadow in the blue quote:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/1aca2386-794f-43f3-aaa2-9fb4a13e8d79)

Commit:
style: :lipstick: Fix styles in hypo page

Fix styles of title in first section, , highlight components in first section, tabs of neonates/children/adults, styles in last section

Commit:
style: :bento: Update assets in adolescents page

Change assets for charts with better resolution and remove unused assets

Commit:
fix: :bug: Remove line that affects other comparators

Remove line that change the color of radiographic footer to white because affects comparators in other pages

Commit:
feat: :construction: Update content in adolescents

Update hero content, update table headings in modal, change chart asset references, change footnotes in radiographics, fix references with italic texts, fix CTA, fix wrappers

Commit:
style: :lipstick: Fix styles on adolescents page

Remove CSS for unused wrappers, add CSS to fix Hero and CTA, Add css for Radiographic footnotes, fix styles of clinical study components, Fix pinch to zoom button position

Commit:
feat: :construction: Update clinical data modal content in Adolescents page

Update content of clinical data modal in adolescents page to Figma v3 and fix style for long paragraph in last cell of table

Commit:
fix: :bug: Fix bug in Hero wrapper for clinical data pages and how it works

Fix bug in Hero wrapper for clinical data pages and how it works

Commit:
style: :bento: Update image assets for tabs in last section

Update image assets that show people in tabs on last section of adults page

Commit:
style: :lipstick: Add new color variable name blue-gray

Add new color variable name blue-gray

Commit:
feat: :lipstick: Add new color variant for Highlight component

Add color variant blue-gray for Highlight component

Commit:
feat: :construction: Update Clinical Data Modal content for Adults Page

Update content to Figma v3, fix styles, add prop to clinical data modal component to add a classname

Commit:
feat: :lipstick: Update content of internal table in Adults page

Update content of internal table in Adults page

Commit:
feat: :lipstick: Fix Adults page

Change layouts of some sections to avoid multiple side spaces (margins and paddings) and to allow custom-titles to extend until border, fix lists with font styles and spaces, Fix Hero, Change colors, font styles and spaces in Highlights components, fix styles for some titles, fix tab in last section, change images in last section

Commit:
fix: :bug: Fix second section in infants for bigger screens

Add wrapper for second section and adjust styles to keep Figma design with new layout

Commit:
fix: :lipstick: Change footnotes for children and adults tabs

Change footnotes for children and adults tabs

Commit:
fix: :construction: Fix some spaces and font-styles in adolescents

Add italic tags for a chart footnote in last section, add max-width for radiographic titles, adjust font-size on mobile for paragraphs and adjust margin-bottom for title in last section on desktop

Commit:
fix: :construction: Fix adults page

Add missing footnote for first section, add missing super index of custom-title in second section, fix some spaces and fix font-styles for footnotes in the last tab sections

HackMort committed:
Hide scrollbar on all tabs

HackMort committed:
Update Hypo page styles

HackMort committed:
Update Style of table on Adults page.

HackMort committed:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/adolescents-update-v3

HackMort reviewed (approved):
I had to do a couple of extra fixes but looks great! Approved!
